src/bot/database/entities/streamer.py

```diff
@@ -0,0 +1,24 @@
+from datetime import datetime
+
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.orm import declarative_base, relationship
+from sqlalchemy.sql.schema import ForeignKey
+from sqlalchemy.sql.sqltypes import TIMESTAMP
+
+Base = declarative_base()
+
+
+class Streamer(Base):
+    __tablename__ = "streamers"
+
+    id = Column(Integer, primary_key=True, autoincrement=True, unique=True)
+    twitch_id = Column(Integer)
+    twitter_id = Column(Integer)
+    name = Column(String)
+    category_id = Column(ForeignKey("categories.id"))
```

Comment:
I think the right thing would be to put the category_id before the name because it is an int and not a varchar.
```suggestion
    category_id = Column(ForeignKey("categories.id"))
    name = Column(String)
```